pkg/volume/lvmlocalpv.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2020-2021 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package volume
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/openebs/openebsctl/pkg/client"
+	"github.com/openebs/openebsctl/pkg/util"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// GetLVMLocalPV returns a list of LVM-LocalPV volumes
+func GetLVMLocalPV(c *client.K8sClient, pvList *corev1.PersistentVolumeList, openebsNS string) ([]metav1.TableRow, error) {
+	var rows []metav1.TableRow
+	var version string
+	if CSIctrl, err := c.GetCSIControllerPod("openebs-lvm-controller"); err == nil {
```

Comment:
Nice :+1:

Reply:
> Instead of pod can we get sts with the label? The sts will have the latest applied configurations and pods can differ. Even if the controller is down to 0 pods the volume will be working fine and the version will remain same as sts.


Are all CSI Controllers STS always?

Reply:
Yes

Reply:
This is the closest one can come to volume/pool(VG) versions for storage entities which lack a data plane, not sure how relevant would this be for debugging purposes.